apicapi/apic_manager.py

```diff
@@ -1464,6 +1327,54 @@ def clear_all_hostlinks(self):
                 except orm.exc.NoResultFound:
                     return
 
+    def _build_config(self, ext_config):
+        if cfg.CONF.apic.apic_password is not None:
```

Comment:
It turns out that there is a valid case when password will not be set, that is when we use certificate based login. Perhaps we can use 'apic_username' instead?